plugins/automation/README.md

```diff
@@ -0,0 +1,4 @@
+# Automation Rules
+
+See the [Insights docs](https://insights.docs.fairwinds.com/release-notes/#_2-2-0) to learn about the Automation rules,
```

Comment:
We probably want to link to an automation page not the release notes

Reply:
We don't have the docs for automation rules yet. Once we add them, we can update this to point to the doc